functions/src/cron.ts

```diff
@@ -0,0 +1,33 @@
+import { firestore } from "firebase-admin";
+import * as functions from "firebase-functions";
+import { db } from "./app";
+
+// TODO: Remove if not testing the logic
+export const testLogic = functions.https.onRequest(async (req, res) => {
+  deleteOldLobbies();
+  res.send({ message: "Hi" });
+});
+
+// Schedule for every day at 3:15AM
+// export const deleteOldLobbies = functions.pubsub.schedule("15 3 * * *").onRun(async (context) => {
+const deleteOldLobbies = async () => {
+  // TODO: Remove if not testing the logic
+  const dayFromTodaySecs = firestore.Timestamp.now().seconds - 86_400; // A day ago from today in seconds
+  const dayFromTodayDate = firestore.Timestamp.fromDate(new Date(dayFromTodaySecs * 1000)); // a day ago from today in date object
```

Comment:
let's call these constants something that if obvious to everone that it refers to the past. maybe we could call it `oneDayAgo` or `twentyFourHoursAgo`